hdf5-vol-daos.spec

```
@@ -72,6 +72,7 @@ HDF5 VOL DAOS with MPICH
 Summary: HDF5 VOL DAOS devel with MPICH
 BuildRequires: hdf5-mpich-devel%{?_isa}
 Requires: hdf5-mpich-devel%{?_isa}
+Requires: %{name}-mpich2
```

```suggestion
Requires: %{name}-mpich2%{?_isa} = %{version}-%{release}
```
is the correct syntax.